Infinispan integration fails on WildFly10

```
Caused by: java.lang.Exception: {"WFLYCTL0288: One or more services were unable to start due to one or more indirect dependencies not being available." => {
    "Services that were unable to start:" => [
        "jboss.deployment.unit.\"camel-infinispan-test.jar\".CdiValidatorFactoryService",
        "jboss.deployment.unit.\"camel-infinispan-test.jar\".WeldStartService",
        "jboss.deployment.unit.\"camel-infinispan-test.jar\".component.InfinispanIntegrationTest.JndiBindingsService",
        "jboss.deployment.unit.\"camel-infinispan-test.jar\".component.InfinispanIntegrationTest.START",
        "jboss.deployment.unit.\"camel-infinispan-test.jar\".component.InfinispanIntegrationTest.WeldInstantiator",
        "jboss.deployment.unit.\"camel-infinispan-test.jar\".component.InfinispanIntegrationTest.WeldInterceptorBindingsService",
        "jboss.deployment.unit.\"camel-infinispan-test.jar\".deploymentCompleteService",
        "jboss.deployment.unit.\"camel-infinispan-test.jar\".jndiDependencyService",
        "jboss.deployment.unit.\"camel-infinispan-test.jar\".moduleDeploymentRuntimeInformationStart",
        "jboss.naming.context.java.comp.camel-infinispan-test.camel-infinispan-test.InfinispanIntegrationTest.env.\"org.wildfly.camel.test.infinispan.InfinispanIntegrationTest\".cacheContainer"
    ],
    "Services that may be the cause:" => [
        "jboss.infinispan.test.config.thread-pool.async-operations",
        "jboss.infinispan.test.config.thread-pool.expiration",
        "jboss.infinispan.test.config.thread-pool.listener",
        "jboss.infinispan.test.config.thread-pool.persistence",
        "jboss.infinispan.test.config.thread-pool.remote-command",
        "jboss.infinispan.test.config.thread-pool.state-transfer",
        "jboss.infinispan.test.config.thread-pool.transport",
        "jboss.infinispan.test.config.transport"
    ]
}}
```

--- a/itests/standalone/extra/src/main/java/org/wildfly/camel/test/infinispan/InfinispanIntegrationTest.java
+++ b/itests/standalone/extra/src/main/java/org/wildfly/camel/test/infinispan/InfinispanIntegrationTest.java
@@ -35,11 +35,6 @@ import org.apache.camel.impl.DefaultCamelContext;
 import org.infinispan.manager.CacheContainer;
 import org.jboss.arquillian.container.test.api.Deployment;
 import org.jboss.arquillian.junit.Arquillian;
-import org.jboss.as.arquillian.api.ServerSetup;
-import org.jboss.as.arquillian.api.ServerSetupTask;
-import org.jboss.as.arquillian.container.ManagementClient;
-import org.jboss.as.controller.descriptions.ModelDescriptionConstants;
-import org.jboss.dmr.ModelNode;
 import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.jboss.shrinkwrap.api.asset.EmptyAsset;
 import org.jboss.shrinkwrap.api.spec.JavaArchive;
@@ -47,12 +42,10 @@ import org.junit.After;
 import org.junit.Assert;
 import org.junit.Test;
 import org.junit.runner.RunWith;
-import org.wildfly.camel.test.common.DMRUtils;
 import org.wildfly.extension.camel.CamelAware;
 
 @Stateless
 @CamelAware
-@ServerSetup({InfinispanIntegrationTest.InfinispanCacheSetupTask.class})
 @RunWith(Arquillian.class)
 public class InfinispanIntegrationTest {
 
@@ -64,23 +57,6 @@ public class InfinispanIntegrationTest {
 
     @Resource(lookup = "java:jboss/infinispan/container/test")
     private CacheContainer cacheContainer;
-
-    static class InfinispanCacheSetupTask implements ServerSetupTask {
-
-        @Override
-        public void setup(final ManagementClient managementClient, String containerId) throws Exception {
-            ModelNode cacheOpAdd = DMRUtils.createOpNode("subsystem=infinispan/cache-container=test", ModelDescriptionConstants.ADD);
-            cacheOpAdd.get("default-cache").set("test");
-            cacheOpAdd.get("jndi-name").set("java:jboss/infinispan/container/test");
-            managementClient.getControllerClient().execute(DMRUtils.createCompositeNode(new ModelNode[]{cacheOpAdd}));
-        }
-
-        @Override
-        public void tearDown(final ManagementClient managementClient, String containerId) throws Exception {
-            ModelNode cacheOpRemove = DMRUtils.createOpNode("subsystem=infinispan/cache-container=test", ModelDescriptionConstants.REMOVE);
-            managementClient.getControllerClient().execute(DMRUtils.createCompositeNode(new ModelNode[]{cacheOpRemove}));
-        }
-    }
 
     @Deployment
     public static JavaArchive deployment() {

--- a/itests/standalone/extra/src/main/resources/configuration/standalone-extra.xml
+++ b/itests/standalone/extra/src/main/resources/configuration/standalone-extra.xml
@@ -266,6 +266,7 @@
                 </local-cache>
                 <local-cache name="timestamps"/>
             </cache-container>
+            <cache-container name="test" default-cache="test" jndi-name="java:jboss/infinispan/container/test"/>
         </subsystem>
         <subsystem xmlns="urn:jboss:domain:jaxrs:1.0"/>
         <subsystem xmlns="urn:jboss:domain:jca:3.0">

